scripts/blogpost.js

```diff
@@ -0,0 +1,163 @@
+// Refactor the blog to generate the blog posts dynamically from JavaScript
+const articleCon = document.getElementById('blog-content');
+
+const post1 = [
+  './images/blog-1.jpg',
+  'A person and their black dog overlooking a canyon',
+  'Traveling With Your Dog',
+  'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.',
+  'Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+];
+
+const post2 = [
+  './images/blog-2.jpg',
+  'Four dogs of different breeds on leashes, sitting down',
+  'How To Walk Multiple Dogs',
+  'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.',
+  'Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+];
+
+const post3 = [
+  './images/blog-3.jpg',
+  'Sihlouettes of girl with a ball in her hand and a dog sitting down, looking at the ball',
+  'Teach Your Dog To Fetch!',
+  'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.',
+  'Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+];
+
+const posts = [
+  'post1',
+  'post2',
+  'post3'
+];
+
+for (let i = 0; i < posts.length; i++) {
```

Comment:
You are very close on this one. Instead of making three arrays (post1, post2, post3), you will want your posts to be javascript objects. Here's an example of one:
```
{
     title: "The Title",
     image: "./images/blog-1.jpg",
     para-one: "The first paragraph text here"
}
```

Now if we turn that into an array:

```
[{
     title: "The Title",
     image: "./images/blog-1.jpg",
     para-one: "The first paragraph text here"
},
{
     title: "The Title",
     image: "./images/blog-1.jpg",
     para-one: "The first paragraph text here"
},
{
     title: "The Title",
     image: "./images/blog-1.jpg",
     para-one: "The first paragraph text here"
}]
```

Then we can loop through the array and access elements like this:

```
for(let i = 0; posts.length; i++) {
let blogTitle = document.createElement("h3");
blogTitle.innerHTML = posts[i].title
}
```